src/basho_bench_duration.erl

```diff
@@ -102,6 +102,17 @@ terminate(Reason, #state{duration=DurationMins}) ->
             end
     end,
     run_hook(basho_bench_config:get(post_hook, no_op)),
+    case basho_bench_config:get(enable_eprof, false) of 
+        false ->
+            ok;
+        true ->
+            ?CONSOLE("Stopping eprof profiling",[]),
+            EprofFile = filename:join(basho_bench:get_test_dir(),"eprof.log"),
+            eprof:stop_profiling(),
+            eprof:log(EprofFile),
+            eprof:analyze(total),
+            ?CONSOLE("Eprof output in ~p\n",[EprofFile])
```

Comment:
Should be spaces after the comma in this line and the other `?CONSOLE` call above.